bindings/nodejs/examples/how_tos/accounts_and_addresses/list-accounts.ts

```diff
@@ -9,7 +9,7 @@ require('dotenv').config({ path: '.env' });
 // Run with command:
 // yarn run-example ./how_tos/accounts_and_addresses/list-accounts.ts
 
-// This example lists all accounts in the wallet.
+// This example lists all account outputs in the wallet.
```

Comment:
Then maybe this example should be moved to the account outputs folder? Doesn't really belong here anymore

Reply:
But the folder still contains `accounts`
Tbh I would wait with moving examples around until we defined a proper structure on the Rust side when we have something working, otherwise we will just move it multiple times

Reply:
Yeah I mean, this example completely changed meaning now, has nothing to do here anymore. But also fine to move later 👍🏻